src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -2,13 +2,20 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-from azure.mgmt.appconfiguration.models import ConfigurationStoreUpdateParameters
+from azure.mgmt.appconfiguration.models import (ConfigurationStoreUpdateParameters,
+                                                ConfigurationStore,
+                                                Sku)
 
 from ._utils import resolve_resource_group, user_confirmation
 
 
-def create_configstore(client, resource_group_name, name, location):
-    return client.create(resource_group_name, name, location.lower())
+def create_configstore(client, resource_group_name, name, location, sku):
```

Comment:
Can you give sku default value None here?
create_configstore(client, resource_group_name, name, location, sku=None)

Reply:
Sku will be a required argument when we announce our pricing model. Since we have ignore this argument in _param.py. I think we should leave it required here.

Reply:
Fixed